src/login/logind-button.c

```diff
@@ -181,6 +220,23 @@ static int button_dispatch(sd_event_source *s, int fd, uint32_t revents, void *u
                         break;
                 }
 
+        } else if (ev.type == EV_KEY && ev.value == 0) {
+                if (ev.code == KEY_RESTART) {
+                        log_struct(LOG_INFO,
+                                   LOG_MESSAGE("Reboot key released."),
+                                   "MESSAGE_ID=" SD_MESSAGE_REBOOT_KEY_RELEASE_STR);
```

Comment:
i think the message here should say "Reboot key pressed short". And I think we should use the same message id as for the key press if handle_reboot_key_long_press is set to HANDLE_IGNORE, i.e. SD_MESSAGE_REBOOT_KEY_STR

I mean, recognizable message id are all great, but let's not get lost in too much detail here. i.e. let's have one message id for short presses and one for long presses, but not more.